### ISSUE TITLE
show result in UE

### ISSUE THREAD

**Issue by luoww1992** [city-super/Octree-GS#26]
i want to show it in UE，but i notice it need original .ply file 
how  to translate the result to original .ply file ?

**Comment by jianglh-WHU** [comment]
Sorry, in the present, the final results from octree-gs cannot be directly converted to the 3D-GS format due to the implementation of the MLP.

**Comment by luoww1992** [comment]
@jianglh-WHU 
if without mlp, it can do ?
if yes, how to do ?

**Comment by ZhiyeTang** [comment]
@luoww1992 Maybe you can try pruning all view-dependent information used in Octree-GS and re-train the code. Afterward, you can use the way as in `generate_neural_gaussians()` to generate all neural Gaussians into a `.ply` file as what it looks like in vanilla 3DGS. Theoretically, it is compatible with the existing UE plugins for 3DGS.

**Comment by ZhiyeTang** [comment]
The fundamental reason why Scaffold-GS derivatives (like Octree-GS and HAC) are difficult to import into UE is because they require generating neural Gaussians on-the-fly every time the view changes. Therefore, without using such view-dependent information, such anchor-based methods can be correctly converted to the original 3DGS format, and loaded into UE like real vanilla 3DGS.

**Comment by jianglh-WHU** [comment]
Yeah! That's a solid move, switching Octree-GS to 3DGS. Just gotta keep a few things in mind:
1. Shift the opacity mask and swap the activation function from tanh to sigmoid when predicting the opacity of the neural Gaussian.
2. It's better to predict spherical harmonics instead of color.
